rules/misc_rules.build_defs

```diff
@@ -602,3 +602,13 @@ def is_platform(os:str|list=CONFIG.OS, arch:str|list=CONFIG.ARCH, host_os:str|li
     host_arch = as_list(host_arch)
 
     return CONFIG.OS in os and CONFIG.ARCH in arch and CONFIG.HOSTOS in host_os and CONFIG.HOSTARCH in host_arch
+
+def link_generated_outs(gen_srcs=CONFIG.LINK_GEN_SOURCES):
```

Comment:
We sure we want to do this in post-build functions? why can't it be done statically?

Reply:
We don't know the outputs until it's built as it's using output directories.